tests/build/test_build_request.py

```diff
@@ -1348,3 +1348,85 @@ def test_prod_non_custom_base_image(self, tmpdir):
         pull_base_image_plugin = get_plugin(
             plugins, 'prebuild_plugins', 'pull_base_image')
         assert pull_base_image_plugin is not None
+
+    def test_render_prod_custom_site_plugin_enable(self):
+        """
+        Test to make sure that when we attempt to enable a plugin, it is
+        actually enabled in the JSON for the build_request post-rendering
+        """
+
+        plugin_type = "exit_plugins"
+        plugin_name = "testing_exit_plugin"
+        plugin_args = {"foo": "bar"}
+
+        build_request = BuildRequest(INPUTS_PATH)
+        build_request.customize_conf['enable_plugins'].append(
+            {
+                "plugin_type": plugin_type,
+                "plugin_name": plugin_name,
+                "plugin_args": plugin_args
+            }
+        )
+        kwargs = get_sample_prod_params()
+        build_request.set_params(**kwargs)
+        build_request.render()
+
+        assert {
+                "name": plugin_name,
+                "args": plugin_args
+        } in build_request.dj.dock_json[plugin_type]
+
+    def test_render_prod_custom_site_plugin_disable(self):
+        """
+        Test to make sure that when we attempt to disable a plugin, it is
+        actually disabled in the JSON for the build_request post-rendering
```

Comment:
'post-rendering' seems like it expects the custom rendering to happen at the end instead of at the start.